### ISSUE TITLE
Political maps

### ISSUE THREAD

**Issue by bkmgit** [echarts-maps/echarts-countries-js#15]
It may be helpful to allow several different world, country and region maps. This would allow end user to choose the one they want rather than make it a developer choice. It would also allow for showing political boundaries at different periods in time.

**Comment by chfw** [comment]
Yes, agree. The organization of these different maps needs consideration.
